polymesh-docs/polymesh-private/confidential-assets/diagrams.md

```diff
@@ -160,7 +160,6 @@ else Venue/Sender does not provide instruction ID
   P -->> R: Pending instruction ID(s) returned
 end
 
-R ->> P: Query previously emitted instruction affirmation<br>events to retrieve sender proof(s).A chain<br>indexer (e.g. Polymesh SubQuery) may be required<br> for efficient queries of historical events
```

Comment:
Revert this change. This step is still relevant even if it is a case that the SDK or REST API can do it in the background it is still being done